src/test/ci/leaderboard.test.ts

```diff
@@ -20,10 +19,10 @@ const HOUR = 6;
 const MINUTE = 5;
 const INITIAL_SECONDS = 3;
 const date = new Date(new Date().getFullYear(), INITIAL_MONTH, INITIAL_DAY, HOUR, MINUTE, INITIAL_SECONDS);
-const secondAgo = new Date(date).setSeconds(INITIAL_SECONDS - 1);
-const yesterday = new Date(date).setDate(INITIAL_DAY - 1);
-const lastWeek = new Date(date).setDate(INITIAL_DAY - 7);
-const lastMonth = new Date(date).setMonth(INITIAL_MONTH - 1);
+const secondAgo = new Date(new Date(new Date(date).setSeconds(INITIAL_SECONDS - 1)));
```

Comment:
Is there an extra new Date here

Reply:
No, `.setSeconds, .setDate, setMonth` all return timestamp in ms